test/framework/framework.go

```diff
@@ -256,7 +256,9 @@ func (f *Framework) CreateNamespace(name string) error {
 		}
 		f.logger.Info("creating namespace", "Namespace", name, "Context", k8sContext)
 		if err := remoteClient.Create(context.Background(), namespace); err != nil {
-			return err
+			if !errors.IsAlreadyExists(err) {
```

Comment:
Are we sure we want this? If we don't anticipate it to exists, perhaps we should exit? There could be some old resources in that case that could cause issues.